WowPacketParser/Store/Objects/PageTextLocale.cs

```diff
@@ -0,0 +1,22 @@
+﻿using WowPacketParser.Loading;
+using WowPacketParser.Misc;
+using WowPacketParser.SQL;
+
+namespace WowPacketParser.Store.Objects
+{
+    [DBTableName("page_text_locale")]
+    public sealed class PageTextLocale : IDataModel
+    {
+        [DBFieldName("ID", true)]
+        public uint? ID;
+
+        [DBFieldName("Text")]
+        public string Text;
+
+        [DBFieldName("locale", true)]
+        public string Locale = BinaryPacketReader.GetClientLocale();
```

Comment:
need something here that doesn't depend on BinaryPacketReader (its not the only reader)